scadm/Cargo.toml

```diff
@@ -18,4 +18,4 @@ macaddr.workspace = true
 
 [dependencies.softnpu]
 git = "https://github.com/oxidecomputer/softnpu"
-branch = "main"
+branch = "nils_test"
```

Comment:
There will be.  Right now that version of softnpu has no code changes, but does have a `Cargo.toml` change pointing at my `p4` branch.  Once I push the `p4` stuff, I will remove the `rev = ` line in softnpu's `Cargo.toml`, and it will get my updated `p4` from `main`.

It's all very circular.